w_titanic

```diff
@@ -0,0 +1,12 @@
+library(tidyr)
+library(dplyr)
```

Comment:
Do you use any functions from these libraries? If not, you could delete this lines.